src/components/themed/SimpleTextInput.tsx

```diff
@@ -27,8 +27,10 @@ import {
   CloseIconAnimated
 } from '../icons/ThemedIcons'
 import { useTheme } from '../services/ThemeContext'
+import { UnscaledText } from '../text/UnscaledText'
+import { UnscaledTextInput } from '../text/UnscaledTextInput'
 
-const AnimatedTextInput = Animated.createAnimatedComponent(TextInput)
+const AnimatedTextInput = Animated.createAnimatedComponent(UnscaledTextInput)
```

Comment:
I suspect this is going to cause perf issues, since we're animating a JS wrapper, and reanimated can't "see" in to realize which props are native or not. Therefore, Reanimated will have to loop Hermes into every frame, which will chunder. I think we need to use `Animated.createAnimatedComponent(TextInput)` directly, and then manually apply `allowFontScaling` to that.